src/cerfitications.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import axios from 'axios';
+
+export type Options = {
+    baseURL: string,
+    auth: string
+};
+
+export default class Certificates {
+    constructor({ auth, baseURL }: Options) {
+        this.client = axios.create({
+            baseURL,
+            headers: {
+                Authorization: auth
+            }
+        });
+    }
+
+    client: typeof axios;
+
+    deleteCertificationID(certificationID : number): Promise<Object> {
```

Comment:
For the resolved data in the promise, it would be good to describe the shape and properties. What does the response object look like?